HerPortal.BusinessLogic/Models/ConsortiumData.cs

```diff
@@ -1,7 +1,13 @@
 ﻿namespace HerPortal.BusinessLogic.Models;
 
+/// <summary>
+///     This class does not contain full Consortium Data.
+///     The full data (including an the Consortium's relationship to LAs) can be found in the HUG2 Public Website codebase
+/// </summary>
 public static class ConsortiumData
 {
+    // If Consortium Codes change, migrations or mapping will be required to maintain
+    // correct relationships with the referenced LA in the HUG2 Public site codebase
```

Comment:
I couldn't quite figure out what this line is trying to tell me, is it that:
- If a consortium code changes in the portal data, a migration (mapping?) will be needed to migrate the database to include the new code
- If a consortium code changes in the main site, the consortium code in this file will need to be updated to reflect that